bundles/ch.elexis.agenda/src/ch/elexis/agenda/ui/AgendaParallel.java

```diff
@@ -87,6 +90,12 @@ protected void create(Composite parent) {
 
 	@Override
 	public void setFocus() {
+		if (isFirstTime) {
+			MessageDialog.openInformation(Display.getDefault().getActiveShell(), "Ansicht veraltet", "Die Ansicht "
+					+ getTitle()
+					+ " ist veraltet und wird nicht mehr unterstützt. Bitte verwenden Sie die Agenda Web Ansicht.");
+			isFirstTime = false;
+		}
```

Comment:
In AgendaWeek (unteres file) werden die messages verwendet, und hier nicht. Bitte hier auch die dafür im core angelegten messages verwenden.